osm2streets/src/marking.rs

```diff
@@ -0,0 +1,134 @@
+//! Semantic definitions of the road markings used to control traffic.
+//!
+//! These types describe the *meaning* of the markings, in a locale-independent way, not necessarily
+//! how they look. See [`crate::paint`] for generating locale-dependant renderings of these markings.
+//!
+//! This initial version of these types is based on, and takes its nomenclature from, the Australian
+//! Manual of Uniform Traffic Control Devices (MUTCD). As such, the distinctions between different
+//! kinds of markings that are represented as variants are tailored to support the marking scheme
+//! used in Australia, not necessarily in other parts of the world. The intention is to expand these
```

Comment:
I like the approach of starting with specific things you're familiar with. Hasty overgeneralization can be harder to back out of